pkg/bootc/bootc_disk.go

```diff
@@ -146,13 +150,25 @@ func (p *BootcDisk) getOrInstallImageToDisk(quiet bool, diskConfig DiskImageConf
 		logrus.Debugf("No existing disk image found")
 		return p.bootcInstallImageToDisk(quiet, diskConfig)
 	}
+	if p.bustCache {
+		logrus.Debug("Found existing disk image but cache busting is enabled, removing and recreating")
+		err = os.Remove(diskPath)
+		if err != nil {
+			return err
+		}
+		return p.bootcInstallImageToDisk(quiet, diskConfig)
+	}
+
 	logrus.Debug("Found existing disk image, comparing digest")
 	defer f.Close()
 	buf := make([]byte, 4096)
 	len, err := unix.Fgetxattr(int(f.Fd()), imageMetaXattr, buf)
 	if err != nil {
 		// If there's no xattr, just remove it
-		os.Remove(diskPath)
+		err = os.Remove(diskPath)
+		if err != nil {
+			return err
+		}
```

Comment:
I don't think we don't want to stop and return an error here if the xattr is not present, just remove the disk file and move on. Maybe we can just log a warning that we found a disk without xattrs

Reply:
if removing the disk here fails then we might get into a weird state later on. Most likely, the rename of the temp disk to the "real" disk will fail because the "real" disk still exists. The error in that case would be misleading.